src/out_queue.rs

```diff
@@ -133,7 +133,7 @@ impl OutQueue {
     pub fn set_their_ack(
         &mut self,
         ack_nr: u16,
-        selective_acks: SmallVec<[u8; 4]>,
+        selective_acks: &SmallVec<[u8; 4]>,
```

Comment:
If we don't need it to be a `SmallVec` then wouldn't it make more sense to just use `&[u8]` here?

Reply:
Good question. I don't know. Seems like `Packet` is the only one producing `SmallVec`: https://github.com/canndrew/tokio-utp/blob/master/src/packet.rs#L231
From what I read about `SmallVec` it seems like it has fits this case: usually selective acks array is 4 bytes size, but if there are some uTP extensions, it will be longer.  I don't know the protocol that well though, so I might be wrong here...